judge.pl

```diff
@@ -881,9 +894,15 @@ sub split_solution {
         }
         $testsets = [ keys %subtasks, @other ? CATS::Testset::pack_rank_spec(@other) : () ];
     }
+    elsif ($r->{split_strategy}->{method} eq $cats::split_explicit) {
+        $testsets = $r->{split_strategy}->{testsets} // [];
+    }
     else {
```

Comment:
This is complicated enough to require tests and documentation.
Please at least consider writing a test.